### ISSUE TITLE
Add support for matches with whitespace to `prosemirror-suggest`

### ISSUE THREAD

**Issue by ifiokjr** [remirror/remirror#353]
## Description

Space characters seem to remove the matching text from being matched by the plugin.

## Checklist

- [x] I have read the [**contributing**](https://github.com/remirror/remirror/blob/master/docs/contributing.md) document.

**Comment by xerosanyam** [comment]
thanks for implementing this.
But I am still unable to understand how the config would look like if we want the activation character to be any character ie `/\S` & want to match whitespace in the mentions list. ex `First Name + " " + Last Name`

**Comment by ifiokjr** [comment]
@xerosanyam I'm not 100% sure what you're trying to accomplish. If you explain, I can send an example, or add the functionality if it doesn't exist.

**Comment by xerosanyam** [comment]
I am sure it'd be simple, just that I am unable to wrap my head around it, right now.

Thanks for your time!

Okay, so the objective is:
1. We need to show mentions popup once user types any character. ie. Trigger on Free flowing text
2. List of options can have whitespace separated items
3. Activation character should not go into "start" mode again once it sees space

For example, we have few users ["Mark Zuckerberg", "Steve Jobs", "Steve Wozniak"]
User types "Steve j", then only Steve jobs should appear.

What I've tried doing is,
I made the activation character to be /\S/
This triggers our suggestions
But once it reaches space after the 1st word it re-triggers & goes into start mode. ie, query becomes empty & I am able to see all the options

In short, our list of options contain whitespace seperated items & we need suggester to not get into "start" mode once it sees space.

**Comment by ifiokjr** [comment]
That's actually a very valid use case. I'll see how I can make it easier to manage it. Off the top of my head I'm not sure how to do it but I'll play around and either add the functionality or send an example. 

Could you create a new issue with the above feature request? I keep missing notifications for closed issues.

I'm also open to API suggestions if you have ideas for how `prosemirror-suggest` could be made improved and simplified.
